client/src/pages/treedata/TreeHeader.js

```diff
@@ -21,13 +22,13 @@ export default function TreeHeader({
         <div>
           <h2><a href={wikipediaLink} name={wikipediaLink} target="_blank" rel="noreferrer">{scientific}</a></h2>
           {(scientific !== genus) && <h2>{genus}</h2>}
-          <h5>Planted: {format(new Date(datePlanted), 'MMMM d, yyyy')}</h5>
-          <h5>Height: {height}</h5>
-          <h5>DBH: {dbh}</h5>
+          {datePlanted && <h5>Planted: {planted}</h5>}
+          {height && <h5>Height: {height}</h5>}
+          {dbh && <h5>DBH: {dbh}</h5>}
         </div>
       )}
 
-      {env !== 'prod'
+      {env !== 'production'
```

Comment:
This `env` string comes from `config.js`, which uses `prod`.  But this check is removed anyway in #119, since editing the tree type works there.